assessment.rb

```diff
@@ -1,12 +1,67 @@
 require 'csv'
 # Assessment answers to:
 
+#ran: rails g model Licensee
+#thenran: rake db:migrate
+
 # question 2:
+Person.create(surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63,weight: 132, dob: '1943-09-26',eye_color: 'Brown', hair_color: 'Black')
```

Comment:
So close! You just created the Licensee model, so why are you using some other model now?